README.md

```diff
@@ -199,6 +199,40 @@ reload your webserver without having permission to do anything else.
 * Don't allow this script to be able to read your domain private key!
 * Don't allow this script to be run as root!
 
+### Example user setup
+
+```
+# create 'acme' user which will run the script
+useradd acme -d /etc/ssl/acme/ -m
+
+# challenges folder writable for the script, readable for the web server
+mkdir /var/www/challenges/
+chown acme:www-data /var/www/challenges/
+chmod 750 /var/www/challenges/
+
+# create an account key readable only by the script
+openssl genrsa 4096 > /etc/ssl/acme/account.key
+chown acme:acme /etc/ssl/acme/account.key
+chmod 400 /etc/ssl/acme/account.key
+
+# create domain key readable only by the web server
+openssl genrsa 4096 > /etc/ssl/acme/domain.key
+chown www-data:www-data /etc/ssl/acme/domain.key
+chmod 400 /etc/ssl/acme/domain.key
+
+# generate your certificate requests readable only by the script
+openssl req ... > /etc/ssl/acme/domain.csr
+chown acme:acme /etc/ssl/acme/domain.csr
+chmod 400 /etc/ssl/acme/domain.csr
+
+# request your keys readable by the script and the web server
+sudo -u acme -H python acme_tiny.py --account-key /etc/ssl/acme/account.key --csr /etc/ssl/acme/domain.csr --acme-dir /var/www/challenges/ > /etc/ssl/acme/signed.crt
+chown acme:www-data /etc/ssl/acme/signed.crt
+chmod 640 /etc/ssl/acme/signed.crt
```

Comment:
So, everybody can retrieve your certificate online, but other users on your server aren't allowed to read the certificate? Why?